eng/native/naming.props

```
@@ -5,25 +5,25 @@
 
   <!-- Add path globs specific to native binaries to exclude unnecessary files from packages. -->
   <Choose>
-    <When Condition="$(PackageRID.StartsWith('win'))">
+    <When Condition="$(PackageRID.StartsWith('win')) or ($(PackageRID.StartsWith('browser')) and '$(HostOS)' == 'windows')">
       <PropertyGroup>
         <ExeSuffix>.exe</ExeSuffix>
         <LibSuffix>.dll</LibSuffix>
         <StaticLibSuffix>.lib</StaticLibSuffix>
         <SymbolsSuffix>.pdb</SymbolsSuffix>
       </PropertyGroup>
     </When>
-    <When Condition="$(PackageRID.StartsWith('osx'))">
+    <When Condition="$(PackageRID.StartsWith('osx')) or $(PackageRID.StartsWith('mac')) or $(PackageRID.StartsWith('ios')) or $(PackageRID.StartsWith('tvos'))">
```

to make it more explicit:

```suggestion
    <When Condition="$(PackageRID.StartsWith('osx')) or $(PackageRID.StartsWith('maccatalyst')) or $(PackageRID.StartsWith('ios')) or $(PackageRID.StartsWith('tvos'))">
```